doc/manpages/keys.asciidoc

```diff
@@ -629,4 +629,14 @@ map <scope> <mode> <key> <keys>
 *keys*::
 	string that lists the keys that will be executed when *key* is hit
 
-Keys are always executed within the *normal* mode.
+All mappings, except those bound to the `insert` mode, will be executed in
```

Comment:
Actually this bit (as well as what I wrote about the `map` command a while ago) is redundant with what's on the `commands` page. I've been thinking for a while now that mapping/unmapping probably deserve their own documentation page, considering the sustancial amount of things that could be said about the subject, what do you think?